### ISSUE TITLE
Adjust/assign heading levels

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#217]
```
Home page
<h2>Limit your search</h2>
<h5 class="panel-title">Facet name</h5>

should be
<h2>Limit your search</h2>
<h3 class="panel-title">Facet name</h3>


Results page
<h4>Limit your search</h4>
<h5 class="panel-title">Facet name</h5>

should be
<h2>Limit your search</h2>
<h3 class="panel-title">Facet name</h3>


Results page
<h2 class="sr-only">Search Results</h2>
<h5 class="index_title col-sm-9 col-lg-10">...

should be
<h2># results</h2>
<h3 class="index_title col-sm-9 col-lg-10">...

(Note h2 should be visible on results page.)

Record view
at top of access panels, add
<h2 class="sr-only">Access</h2>

<div class="panel-heading">Available online</div>

should be
<div class="panel-heading"><h3>Available online</h3></div>

<div class="library-location-heading-text">
            <a href="http://library.stanford.edu/libraries/branner/about">Earth Sciences Library (Branner)</a>
            <div class="small location-hours-today">
            </div>
          </div>

should be
<div class="library-location-heading-text">
         <h3>   <a href="http://library.stanford.edu/libraries/branner/about">Earth Sciences Library (Branner)</a> </h3>
            <div class="small location-hours-today">
            </div>
          </div>
```
